changelog.md

```diff
@@ -115,739 +117,3 @@
 - [#3729](https://github.com/ignite/cli/pull/3729) Fix broken generator due to caching /tmp include folders
 - [#3767](https://github.com/ignite/cli/pull/3767) Fix `v0.50` ibc genesis issue
 - [#3808](https://github.com/ignite/cli/pull/3808) Correct TS code generation to generate paginated fields
-
-## [`v0.27.2`](https://github.com/ignite/cli/releases/tag/v0.27.2)
```

Comment:
Why remove the changelog version history?

Reply:
Because it's from the release branch, not in main. Release branch only to track their own version, while main branch tracks everything.